core/src/main/java/org/commonjava/maven/ext/core/impl/BaseGroovyManipulator.java

```diff
@@ -69,6 +71,17 @@
 
     public abstract int getExecutionIndex();
 
+    String fixFileURL( final String script )
```

Comment:
This means this method will be exposed to even the Gradle Manipulator tooling which is not ideal - given its internal only. Can we discuss offline please.